README.md

```diff
@@ -1 +1,77 @@
-# mobile-portfolio
\ No newline at end of file
+# Portfolio
+
+The project is inspired by the figma microverse template 1 (https://figma.com/)
+The goal of this project is to learn how parse a Figma design to create a UI, and use Flexbox to place elements in the page.
+
+
+
+- Screenshoot I
+
+![screenshot](./image/mob1.PNG)
+
+
+- Screenshoot II
+![screenshot](./image/mob2.PNG)
+
+
+- Screenshoot III
+![screenshot](./image/mob3.PNG)
+
+
+- set up a new repository and prepare it for development using best practices.
+- build mobile website.
+ 
+
+## Live Demo
+
+[Live Demo Link](https://http://127.0.0.1:5500/index.html/)
+ 
```

Comment:
- [ ] Kindly, add the live link demo which working properly.
![livelink](https://user-images.githubusercontent.com/55635977/126406149-c7c156bd-bd7d-486d-8a88-c9f49130c91c.png)